kubernetes/deployment.template.yaml

```diff
@@ -0,0 +1,107 @@
+# Example Kubernetes resource template
+#
+# All variables are replaced at deployment time with environment variables.
+#
+# The following variables are derived as follows by the pipeline. DO NOT CHANGE:
+#
+#   - NAMESPACE:    product
+#   - SERVICE_NAME: product-component-branch
+#   - IMAGE_NAME:   registry-name/hmcts/product-component:branch
+#
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    name: ${SERVICE_NAME}
+    namespace: ${NAMESPACE}
+spec:
+    replicas: 1
+    selector:
+        matchLabels:
+            app: ${SERVICE_NAME}
+    template:
+        metadata:
+            labels:
+                app: ${SERVICE_NAME}
+        spec:
+            dnsConfig:
+                options:
+                - name: single-request-reopen
+            containers:
+            - image: ${IMAGE_NAME}
+              name: ${SERVICE_NAME}
+                # These are obtained from environment variables.  If it's
+                # an environment variable, it can be used here.  Projects
+                # would usually source custom values from:
+                #   - Jenkins Azure Keyvault plugin (e.g. secrets)
+                #   - Your project's Jenkinsfile
+              env:
+              - name: REFORM_TEAM
+                value: 'cmc'
```

Comment:
?

Reply:
:sweat_smile: yer maybe should just ask you to review once im finished cleaning it up correctly